### ISSUE TITLE
Contribution contact update

### ISSUE THREAD

**Issue by rkazak** [arsenetar/dupeguru#941]
Hi,

On the web page - https://dupeguru.voltaicideas.net/help/en/contribute.html - under the "let me know" link,
mail is being sent to the original author at "mailto:hsoft@hardcoded.net".  I guess that should be updated?

One of my questions was do you have a platform for general discussions i.e. Discord or Slack or ... ?

**Comment by arsenetar** [comment]
Yes the contact information there should probably be updated, there are other bits of information in the help that are also a bit outdated as well.
There is currently not a Discord or Slack,, although I am not opposed to having something like that setup.  I have enabled github's discussions feature that recently became available.

**Comment by pantropia** [comment]
Sorry just seen this after raising a new issue for the same thing.
